c7n/resources/apigw.py

```diff
@@ -250,6 +302,28 @@ def process_task_set(self, client, task_set):
 
 @RestResource.action_registry.register('update-method')
 class UpdateRestMethod(BaseAction):
+    """Change or remove api method behaviors based on key value
+
+    :example:
+
+    .. code-block: yaml
+
+        policies:
+          - name: enforce-iam-permissions-on-api
+            resource: rest-resource
+            filters:
+              - type: rest-method
+                key: authorizationType
+                value: NONE
+                op: eq
+            actions:
+              - type: update-method
+                patch:
+                  -
```

Comment:
formatting looks off

Reply:
Yeah, I think I can start with `- op:` and continue the dict below... i just got used to writing lists of dict in yaml like that.